docs/releases/2.8.0.rst

```diff
@@ -149,6 +149,8 @@ Changes in settings
   This restricts how many units are shown in export views.
 - :setting:`POOTLE_SEARCH_BACKEND` was added, to allow configuring the search
   backend to be used.
+- Changed the default value for ``ACOUNT_SESSION_REMEMBER`` so now sessions are
```

Comment:
Lets actually link to the upstream docs if they exist https://django-allauth.readthedocs.io/en/latest/configuration.html